### PR TITLE
Boilerplate code for serving Merle via Flask

### PR DESCRIPTION
**Pull request by Hevia**

### CODE REVIEW

File: src/app/server.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+from flask import flask
+
+app = Flask(__name__)
+
+
```
Comment (lloydoad) [inline src/app/server.py:5]: nit: double line?

File: src/app/server.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+@app.route("/ping")
+def pong():
+    return "Merle is alive!"
+
+
```
Comment (lloydoad) [inline src/app/server.py:10]: nit: double line?

File: src/app/server.py [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+if __name__ == "__main__":
+    app.run(host="0.0.0.0")
```
Comment (lloydoad) [inline src/app/server.py:12]: Nit: maybe store "0.0.0.0" in named variable/constant? Helps with maintenance overtime

File: src/app/server.py [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+
+app = Flask(__name__)
+
+@app.route("/ping")
+def pong():
+    return "Merle is alive!"
+
+if __name__ == "__main__":
+    _HOST = "0.0.0.0"
+    _PORT = 5001
```
Comment (lloydoad) [inline src/app/server.py:11]: Nit: maybe a personal style choice but consider moving constants to top of file? around lines 2-4. I'm guessing if more stuff get added to this file, the intuition would be to check the top for what the definition of stuff are.

File: src/poetry.lock [hunk 5]
```diff
@@ -0,0 +1,130 @@
+[[package]]
```
Comment (lloydoad) [inline src/poetry.lock:1]: I'm basing this off npm packages but are `.lock` files suppose to be uploaded to the repo? Won't this cause an issue where we are downloading outdated dependencies when we try to re-install on a different device? I could be entirely off here though; basing this on package.lock knowledge.

File: src/pyproject.toml [hunk 6]
```diff
@@ -0,0 +1,15 @@
+[tool.poetry]
+name = "Merle"
+version = "0.1.0"
+description = ""
+authors = ["Your Name <you@example.com>"]
```
Comment (lloydoad) [inline src/pyproject.toml:5]: Nit: template email

File: src/poetry.lock (hunk 5 above)
Comment (Hevia) [reply src/poetry.lock:1]: totally missed this, thanks for the catch

**Pull request merged**